lib/msal-node/src/client/ClientApplication.ts

```diff
@@ -162,8 +162,8 @@ export abstract class ClientApplication {
         } catch (e) {
             if (e instanceof AuthError) {
                 e.setCorrelationId(validRequest.correlationId);
+                serverTelemetryManager.cacheFailedRequest(e);
```

Comment:
`serverTelemetryManager.cacheFailedRequest(e)` expect an instance of `AuthError`.

Reply:
If `serverTelemetryManager.cacheFailedRequest()` is able to accept any types of error, do these need to be reverted? (and below)

Reply:
I have already reverted it but forgot to resolve the comment. Sorry about that.

Reply:
But `cacheFailedRequest` is still being called instide the `if (e instanceof AuthError)` block

Reply:
Completely forgot about `msal-node` module. Good catch, thank you. Fixed.